test/parallel/test-child-process-kill.js

```diff
@@ -39,3 +39,41 @@ assert.strictEqual(cat.signalCode, null);
 assert.strictEqual(cat.killed, false);
 cat.kill();
 assert.strictEqual(cat.killed, true);
+
+// Test different types of kill signals on Windows.
+if (common.isWindows) {
+  const process1 = spawn('cmd');
```

Comment:
Can this be simplified down to a for loop of test cases?

Reply:
Thank you for the review. Fixed it.